packages/react/src/ButtonGroup/ButtonGroup.module.css

```
@@ -0,0 +1,8 @@
+.ButtonGroup {
+  display: inline-grid;
+  gap: 8px;
```

We have an equivalent design token for this.. can we switch to that please? 

```suggestion
  gap: var(--base-size-8);
```